docs/core/rid-catalog.md

```diff
@@ -76,137 +80,66 @@ There are some considerations about RIDs that you have to keep in mind when work
-  - `rhel.6-x64`
-- Tizen
-  - `tizen`
-  - `tizen.4.0.0`
-  - `tizen.5.0.0`
-  - `tizen.5.5.0`
-  - `tizen.6.0.0`
-  - `tizen.6.5.0`
-  - `tizen.7.0.0`
+To be able to use RIDs, you have to know which RIDs exist. For the latest and complete version, see the [PortableRuntimeIdentifierGraph.json](https://github.com/dotnet/sdk/blob/main/src/Layout/redist/PortableRuntimeIdentifierGraph.json) in the [`dotnet/sdk`](https://github.com/dotnet/sdk) repository.
+
+RIDs that are considered 'portable' - that is, aren't tied to a specific version or OS distribution - are the recommended choice. This means that portable RIDs should be used for both [building a platform-specific application](./deploying/index.md#platform-specific-and-framework-dependent) and [creating a NuGet package with RID-specific assets](/nuget/create-packages/supporting-multiple-target-frameworks#architecture-specific-folders).
+
+Starting with .NET 8, the default behaviour of the .NET SDK and runtime is to only consider non-version-specific and non-distro-specific RIDs. When restoring and building, the SDK [uses a smaller, portable RID graph](https://github.com/dotnet/docs/issues/36527). The <xref:System.Runtime.InteropServices.RuntimeInformation.RuntimeIdentifier?displayProperty=nameWithType> [returns the platform for which the runtime was build](https://github.com/dotnet/docs/issues/36466). At run time, .NET will find [RID-specific assets using a known set of portable RIDs](./compatibility/deployment/8.0/rid-asset-list.md). When building an application with RID-specific assets that may be ignored at runtime, the SDK will emit a warning: [NETSDK1206](./tools/sdk-errors/netsdk1206.md).
+
+### Loading assets for a specific OS version or distribution
+
+.NET no longer attempts to provide first-class support for resolving dependencies that are specific to an OS version or distribution. If your application or package needs to load different assets based on OS version or distribution, it should implement the logic to conditionally load assets.
+
+To get information about the platform, use <xref:System.OperatingSystem?displayProperty=nameWithType> APIs. On Windows and macOS, <xref:System.Environment.OSVersion?displayProperty=nameWithType> will [return the operating system version](./compatibility/core-libraries/5.0/environment-osversion-returns-correct-version.md). On Linux, it may be the kernel version &mdash; to get the Linux distro name and version information, the recommended approach is to read the */etc/os-release* file.
+
+.NET provides various extension points for customizing loading logic &mdash; for example, <xref:System.Runtime.InteropServices.NativeLibrary.SetDllImportResolver(System.Reflection.Assembly,System.Runtime.InteropServices.DllImportResolver)?displayProperty=nameWithType> and <xref:System.Runtime.Loader.AssemblyLoadContext.ResolvingUnmanagedDll?displayProperty=nameWithType>. These can be used to load the asset corresponding to the current platform.
```

Comment:
We should also include extension points for loading managed assemblies - it's not common, but for completeness.